zipkin-server/src/main/java/zipkin/server/ZipkinServer.java

```diff
@@ -13,17 +13,70 @@
  */
 package zipkin.server;
 
+import com.nike.riposte.server.Server;
+import com.nike.riposte.server.config.ServerConfig;
+import com.nike.riposte.server.http.Endpoint;
+import com.nike.riposte.server.http.filter.RequestAndResponseFilter;
+import com.nike.riposte.server.http.filter.impl.AllowAllTheThingsCORSFilter;
+
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ConfigurableApplicationContext;
+
+import java.io.IOException;
+import java.security.cert.CertificateException;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
 import zipkin.server.brave.BootstrapTrace;
 
+import static java.util.Collections.singletonList;
+
 @SpringBootApplication
 @EnableZipkinServer
 public class ZipkinServer {
 
-  public static void main(String[] args) {
-    new SpringApplicationBuilder(ZipkinServer.class)
+  public static void main(String[] args) throws CertificateException, InterruptedException, IOException {
+    ConfigurableApplicationContext cac = new SpringApplicationBuilder(ZipkinServer.class)
```

Comment:
I have no idea what it would take to make Riposte a transport option for Spring Boot, especially since Riposte isn't a Servlet architecture. 

And on the rest this is where my inexperience with the Zipkin ecosystem means I have to rely on you for guidance - I could do a 100% conversion of zipkin-server to Riposte just treating it bindly as a generic API and converting endpoints, but there's still a good chance I'd be missing something important in terms of features or compatibility with the other stuff out there.